go/test/endtoend/vtorc/readtopologyinstance/main_test.go

```diff
@@ -105,7 +105,7 @@ func TestReadTopologyInstanceBufferable(t *testing.T) {
 	assert.Equal(t, primaryInstance.ReplicationSQLThreadState, inst.ReplicationThreadStateNoThread)
 
 	// insert an errant GTID in the replica
-	_, err = utils.RunSQL(t, "insert into vt_insert_test(id, msg) values (10173, 'test 178342')", replica, "vt_ks")
+	_, err = utils.RunSQLWithSuperReadOnly(t, "insert into vt_insert_test(id, msg) values (10173, 'test 178342')", replica, "vt_ks", true)
```

Comment:
When intended to generate an errant GTID, using that function makes sense. 🙂

Reply:
Thats the only place where we pass true to super-read-flag parameter in this function